internal/payloads/error_payload.go

```diff
@@ -63,6 +55,19 @@ func NewInvalidRequestError(ctx context.Context, err error) *ResponseError {
 	}
 }
 
+func NewMissingRequestParameterError(ctx context.Context, param string) *ResponseError {
+	msg := fmt.Sprintf("missing parameter: %s", param)
+	if logger := ctxval.Logger(ctx); logger != nil {
+		logger.Error().Msg(msg)
```

Comment:
Do we want to log it as error? It is not an application error, it is client error, so we shall log it to Warn or maybe even just info IMHO :)